test/tests/auto/Forge/ForgeMultipleModulesExportTest.php

```diff
@@ -0,0 +1,406 @@
+                                        )
+
+                                    [9] => Aternos\Codex\Log\Line Object
+                                        (
+                                            [text:protected] => 	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.0/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:60)
+                                            [number:protected] => 14
+                                        )
+
+                                    [10] => Aternos\Codex\Log\Line Object
+                                        (
+                                            [text:protected] => 	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.0/cpw.mods.modlauncher.Launcher.run(Launcher.java:104)
+                                            [number:protected] => 15
+                                        )
+
+                                    [11] => Aternos\Codex\Log\Line Object
+                                        (
+                                            [text:protected] => 	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.0/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
+                                            [number:protected] => 16
+                                        )
+
+                                    [12] => Aternos\Codex\Log\Line Object
+                                        (
+                                            [text:protected] => 	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.0/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
+                                            [number:protected] => 17
+                                        )
+
+                                    [13] => Aternos\Codex\Log\Line Object
+                                        (
+                                            [text:protected] => 	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.0/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
+                                            [number:protected] => 18
+                                        )
+
+                                    [14] => Aternos\Codex\Log\Line Object
+                                        (
+                                            [text:protected] => 	at cpw.mods.bootstraplauncher@1.0.0/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149)
+                                            [number:protected] => 19
+                                        )
+
+                                )
+
+                            [level:protected] => WARN
+                            [time:protected] => 
+                            [iterator:protected] => 0
+                            [prefix:protected] => [20:14:29] [main/WARN] [ne.mi.fm.lo.FMLConfig/CORE]:
+                        )
+
+                    [counter:protected] => 1
+                    [solutions:protected] => Array
```

Comment:
There should be two solutions suggested here, either the test isn't updated or something else is wrong.